app/app/controllers/cbv_flows_controller.rb

```diff
@@ -36,7 +36,13 @@ def summary
   end
 
   def share
+    if params[:commit] == I18n.t("cbv_flows.share.share_with_caseworker")
+      # replace with actual caseworker_email e.g. to_email = params[:cbv_flow][:caseworker_email]
+      to_email = ENV["SLACK_TEST_EMAIL"]
+    pdf_path = "#{Rails.root}/tmp/#{params[:id]}.pdf"
+    ApplicantMailer.send_pdf_to_applicant(to_email, pdf_path).deliver_now
```

Comment:
If we are going to use a temporary file, let's use Ruby's [Tempfile#create](https://ruby-doc.org/stdlib-3.1.1/libdoc/tempfile/rdoc/Tempfile.html#method-c-create) so that this file gets cleaned up after we're done sending.

```suggestion
    Tempfile.create("cbv-summary-#{params[:id]}", Rails.root.join("tmp")) do |f|
      # <placeholder for rendering the PDF into f.path>
      ApplicantMailer.send_pdf_to_applicant(to_email, f.path).deliver_now
    end
```

But actually after writing this comment, I found the documentation for WickedPdf that shows a one-liner for attaching a rendered PDF to an email: https://github.com/mileszs/wicked_pdf?tab=readme-ov-file#include-in-an-email-as-an-attachment
^ Check that out. Maybe we actually should just do that in the ApplicantMailer and not even worry about passing temporary files around. The PDFs would be stored entirely in memory, which could be inefficient for big PDFs, but I think these ones should be pretty small.